plugin-bazel/src/main/kotlin/org/jetbrains/bazel/languages/starlark/psi/expressions/arguments/StarlarkArgumentExpression.kt

```diff
@@ -0,0 +1,15 @@
+package org.jetbrains.bazel.languages.starlark.psi.expressions.arguments
+
+import com.intellij.lang.ASTNode
+import com.intellij.psi.PsiReference
+import org.jetbrains.bazel.languages.starlark.psi.StarlarkBaseElement
+import org.jetbrains.bazel.languages.starlark.psi.StarlarkElementVisitor
+import org.jetbrains.bazel.languages.starlark.references.StarlarkArgumentReference
+
+class StarlarkArgumentExpression(node: ASTNode) :
+  StarlarkBaseElement(node),
+  StarlarkArgumentElement {
+  override fun acceptVisitor(visitor: StarlarkElementVisitor) = visitor.visitArgumentExpression(this)
+
+  override fun getReference(): PsiReference = StarlarkArgumentReference(this, textRange)
```

Comment:
**suggestion (performance):** Consider caching the reference

If getReference() is likely to be called multiple times, consider caching the StarlarkArgumentReference to avoid creating new objects repeatedly. Ensure to handle the lifecycle properly to avoid stale references.